opentelemetry-user-events-logs/benches/logs.rs

```diff
@@ -0,0 +1,89 @@
+/*
+    The benchmark results:
+    criterion = "0.5.1"
+
+    Hardware: Apple M4 Pro
+    Total Number of Cores:	10
+    // When no listener
+    | Test                        | Average time|
+    |-----------------------------|-------------|
+    | User_Event_4_Attributes     | 10 ns       |
+    | User_Event_6_Attributes     | 10 ns       |
+
+    // When listener is enabled
+    | Test                        | Average time|
+    |-----------------------------|-------------|
+    | User_Event_4_Attributes     | 10 ns       |
```

Comment:
Same results as when listener is enabled?

Reply:
updated now. Please check.